controller_manager/src/controller_manager.cpp

```diff
@@ -188,7 +188,7 @@ ControllerManager::ControllerManager(
   const std::string & node_namespace, const rclcpp::NodeOptions & options)
 : rclcpp::Node(manager_node_name, node_namespace, options),
   resource_manager_(std::make_unique<hardware_interface::ResourceManager>(
-    update_rate_, this->get_node_clock_interface())),
+    this->get_node_clock_interface())),
```

Comment:
@destogl Why is this update_rate removed from the constructor of the `ResourceStorage` and set separately initialized later?.

Reply:
I decided to go that way because now we actually read update rate from the prameters and then setting us in the ResourceStorage. Before one could potentially end up with the standard 100 Hz set even the parameter is something else.

Reply:
Setting this in constructor is wrong in my oppinioin, beucase at the moment when we are constructing ResouceManager we don't know it yet, as parameter is not read. Additionally, this is imporant for simulators that extend ResouceManger to enable custom compoent impolementations.

Reply:
I understand now! Make sense that we might not be able to know it at the construction time of the resource manager, and be able to set it up later. Thank you for explaining it. 

You can resolve this now